<ui version="4.0" >
 <class>UserFitFunctionDialog</class>
 <widget class="QDialog" name="UserFitFunctionDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="bboxOKCancel" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>500</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons" >
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeFunctions" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>20</y>
     <width>441</width>
     <height>192</height>
    </rect>
   </property>
   <property name="midLineWidth" >
    <number>0</number>
   </property>
   <property name="editTriggers" >
    <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
   </property>
   <property name="headerHidden" >
    <bool>false</bool>
   </property>
   <property name="expandsOnDoubleClick" >
    <bool>false</bool>
   </property>
   <column>
    <property name="text" >
     <string>Functions</string>
    </property>
   </column>
   <column>
    <property name="text" >
     <string>Expression</string>
    </property>
   </column>
   <item>
    <property name="text" >
     <string>Background</string>
    </property>
    <property name="text" >
     <string/>
    </property>
    <item>
     <property name="text" >
      <string>Linear</string>
     </property>
     <property name="text" >
      <string>y0+a*x</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Quadratic</string>
     </property>
     <property name="text" >
      <string>y0+a*x+b*x^2</string>
     </property>
    </item>
   </item>
   <item>
    <property name="text" >
     <string>Buit-in</string>
    </property>
    <property name="text" >
     <string/>
    </property>
    <item>
     <property name="text" >
      <string>Boltzmann</string>
     </property>
     <property name="text" >
      <string>A2+(A1-A2)/(1+exp((x-x0)/dx))</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>ExpDecay</string>
     </property>
     <property name="text" >
      <string>y0+A*exp(-x/t)</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>ExpGrowth</string>
     </property>
     <property name="text" >
      <string>y0+A*exp(x/t)</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Gaussian</string>
     </property>
     <property name="text" >
      <string>h*exp( -((x-x0 )/ w)^2)</string>
     </property>
     <item>
      <property name="text" >
       <string>Peak</string>
      </property>
      <property name="text" >
       <string>x0,h,w</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Width</string>
      </property>
      <property name="text" >
       <string>w/1.53454</string>
      </property>
     </item>
    </item>
    <item>
     <property name="text" >
      <string>Lorentzian</string>
     </property>
     <property name="text" >
      <string>h*w^2/((x-x0)^2+w^2)</string>
     </property>
     <item>
      <property name="text" >
       <string>Peak</string>
      </property>
      <property name="text" >
       <string>x0,h,w</string>
      </property>
     </item>
     <item>
      <property name="text" >
       <string>Width</string>
      </property>
      <property name="text" >
       <string>w/2</string>
      </property>
     </item>
    </item>
   </item>
  </widget>
  <widget class="QPlainTextEdit" name="teExpression" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>280</y>
     <width>441</width>
     <height>141</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAdd" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>260</y>
     <width>46</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>Function</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>430</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Peak parameters</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePeakParams" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>430</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>460</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Width formula</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leWidthFormula" >
   <property name="geometry" >
    <rect>
     <x>100</x>
     <y>460</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMultiply" >
   <property name="geometry" >
    <rect>
     <x>90</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Multiply</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsert" >
   <property name="geometry" >
    <rect>
     <x>170</x>
     <y>220</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Insert</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bboxOKCancel</sender>
   <signal>accepted()</signal>
   <receiver>UserFitFunctionDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bboxOKCancel</sender>
   <signal>rejected()</signal>
   <receiver>UserFitFunctionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
